config/app.default.php

```diff
@@ -264,6 +271,7 @@
             'quoteIdentifiers' => false,
             'log' => false,
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            'url' => env('TEST_DATABASE_URL', null),
```

Comment:
Maybe `DATABASE_TEST_URL` would be more consistent with other configs, and to see directly this is about DATABASE ?